packages/apollo-cli/src/commands/schema/download.ts

```diff
@@ -35,6 +35,10 @@ export default class SchemaDownload extends Command {
       description:
         "The URL of the server to fetch the schema from or path to ./your/local/schema.graphql"
     }),
+    insecure: flags.boolean({
```

Comment:
Wondering about the naming, because it's a bit confusing to have `insecure`, `skipsSSLValidation `, and `rejectUnauthorized` referring to the same option. 

`skipSSLValidation` seems the most descriptive (I prefer skip vs. skips, but that's no biggie). I understand wanting to follow `curl` in using `insecure`, but consistency between command flag and config is more important here I think.

Reply:
I agree on this that giving a more self-explanatory name is more important than keeping old names. So I decided to change this (also to skip instead of skips). Thanks for the feedback